microsoft-edge/devtools/settings/throttling.md

```diff
@@ -0,0 +1,123 @@
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: devtools
+ms.date: 06/12/2025
+---
+<!-- Copyright Sofia Emelianova
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       https://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.  -->
+# Throttling
+<!-- https://developer.chrome.com/docs/devtools/settings/throttling -->
+
+In the **Throttling** page of **Customize and control DevTools** (![Customize and control DevTools icon](./throttling-images/customize-and-control-devtools-icon.png)) > **Settings**, you can:
+
+* Calibrate CPU throttling presets to approximate the typical performance of low- and mid-tier mobile devices.
+
+* Set up custom throttling profiles.  You can use your custom throttling profiles to test custom connection speeds in the **Network** tool.<!--, per [Create a custom throttling profile](../network/reference.md#create-a-custom-throttling-profile) in _Network features reference_.  todo: after https://github.com/MicrosoftDocs/edge-developer/pull/3405 is merged, link -->
+
+![The Throttling tab in DevTools Settings](./throttling-images/settings-throttling.png)
+
+
+<!-- ====================================================================== -->
+## Calibrate CPU throttling presets
+<!-- https://developer.chrome.com/docs/devtools/settings/throttling#cpu-throttling -->
+
+To calibrate CPU throttling presets:
+
+1. Go to a webpage, such as the [Peer connection](https://webrtc.github.io/samples/src/content/peerconnection/pc1/) WebRTC sample.
+
+1. Right-click the webpage and then select **Inspect**.
+
+   DevTools opens.
+
+1. In DevTools, click the **Customize and control DevTools** (![Customize and control DevTools icon](./throttling-images/customize-and-control-devtools-icon.png)) button, and then select **Settings** (![Settings icon](./throttling-images/settings-icon.png)).
+
+   The DevTools **Settings** page opens.
+
+1. On the left, select the **Throttling** settings page.
+
+1. In the **CPU throttling presets** section, click the **Calibrate** button, and then click the **Continue** button.
+
+   After about 5 seconds, DevTools navigates away from the current webpage and then reloads it.
+
+   The **Throttling** tab shows the CPU throttling rates that you can apply to your device to get an idea of how your page performs on low- and mid-tier mobile devices:
+
+   ![Calibrated rates next for low- and mid-tier mobile devices](./throttling-images/calibrated.png)
+
+1. In DevTools **Settings**, click the **Close** (X) button.
```

Comment:
We normally use an inline image for icons. You could create the icon by copying this image: `/edge-developer/microsoft-edge/devtools/network/index-images/close-icon.png`.

Reply:
<img width="328" height="21" alt="image" src="https://github.com/user-attachments/assets/f2b2f26e-c270-4714-981b-a8a649e8e3f8" />